src/components/OwnerPanel/styles.module.scss

```diff
@@ -88,3 +88,19 @@ $container-size: 160px;
   background-color: #dd678a;
   border: 1px solid #dd678a;
 }
+
+p {
+  padding-bottom: 1px;
+  line-height: 1.75em;
+  letter-spacing: 0.015em;
+}
+
+.extraInfoKey{
+  font-weight: bold
+}
+
+.extraInfoValue{
+  font-weight:normal
```

Comment:
nit needs a space after this. Before you merge, can you run `yarn format` and `yarn eslint --fix .`

Reply:
done